eng/pipelines/common/xplat-setup.yml

```diff
@@ -115,8 +115,10 @@ jobs:
           queue: BuildPool.Ubuntu.1604.Amd64
 
         # OSX Build Pool (we don't have on-prem OSX BuildPool
-        ${{ if in(parameters.osGroup, 'OSX', 'iOS', 'tvOS') }}:
+        ${{ if and(in(parameters.osGroup, 'OSX', 'iOS', 'tvOS'), ne(parameters.archType, 'arm64')) }}:
           vmImage: 'macOS-10.14'
+        ${{ if and(in(parameters.osGroup, 'OSX', 'iOS', 'tvOS'), eq(parameters.archType, 'arm64')) }}:
+          vmImage: 'macOS-10.15'
```

Comment:
would it be easier to move everything to use `10.15` or we want only arm64 builds on that version?

Reply:
I'll let @dotnet/runtime-infrastructure make that call.  I didn't want to affect other pipelines unnecessarily.

Reply:
Is osx backwards compatible? I mean if we build in macOS-10.15, can the product still run in 10.14? cc: @wfurt

Reply:
The build is backward compatible. I've run official builds on 10.13 without an issue for example. And I prefer consistency, otherwise things get hard to track. People who contribute are probably on 10.15 already given the Apple tendency is to stay on latest and 11 is not far away.

Reply:
Our cmake scripts can/should specify the minimum compatible osx target at compile time.  Then the build should be fully backward compatible.

I'll can make the change.  If we need to revert it we have plany of time before 6.0 ships :wink:

Reply:
Sounds good to me.

Reply:
There is risk that building on newer OS can pick up API not available on older versions. Also for 11.0 used SDK version changes outcome of some functions. However, I think the risk is small at the movement.

Reply:
@wfurt, we pass `-mmacosx-version-min=10.13` to the compiler, so no matter which OS we build on, the code should be runnable on 10.13+.